concordia/signals/handlers.py

```diff
@@ -33,3 +40,25 @@ def update_asset_status(sender, *, instance, **kwargs):
     instance.asset.transcription_status = new_status
     instance.asset.full_clean()
     instance.asset.save()
+
+
+@receiver(post_save, sender=Asset)
+def send_asset_update(sender, *, instance, **kwargs):
+    channel_layer = get_channel_layer()
+    AsyncToSync(channel_layer.group_send)(
+        "asset_updates",
+        {
+            "type": "asset_update",
+            "asset_slug": instance.slug,
```

Comment:
Slugs aren't necessarily unique across items — I'm thinking this should either be the PK or the URL so the client can match it easily. The URL would be more proper but the PK doesn't require looking up the campaign, project, and item values and that's already in our API response for that very reason.

```suggestion
            "id": instance.pk,
```